modules/streamlit-app.py

```diff
@@ -53,6 +53,22 @@
 with col5:
     col5.caption("Deployment")
 
+# preloading options
+if "image_list_mlflow" not in st.session_state:
+    st.session_state["image_list_mlflow"] = inf_runner.run_coroutine(
+        inf_runner.list_images(github=True, platform=True)
+    )
+if "image_list_triton" not in st.session_state:
+    st.session_state["image_list_triton"] = inf_runner.run_coroutine(
+        inf_runner.list_images(triton=True, platform=True)
+    )
+if "image_tags" not in st.session_state:
+    st.session_state["image_tags"] = {}
+if "presets" not in st.session_state:
+    st.session_state["presets"] = inf_runner.run_coroutine(
+        inf_runner.list_preset_names()
+    )
```

Comment:
we could utilize asyncio to create tasks and `gather` them in parallel to speedup the process even more, if you like :)

Reply:
but since this is just a streamlit thing, we could leave it as is not to write more code that will be thrown away :)